content/en/docs/kubernetes/collector/components.md

```diff
@@ -661,3 +663,41 @@ subjects:
     name: otel-collector-opentelemetry-collector
     namespace: default
 ```
+
+## Prometheus Receiver
+
+Prometheus is a common format for Kubernetes metrics. The Prometheus receiver is
```

Comment:
By kubernetes metrics, do you mean metrics from kubernetes components (e.g. kubelet), or metrics from applications running on k8s?  If the former, https://kubernetes.io/docs/reference/instrumentation/metrics/ may be helpful to link to.

Reply:
I'm not a Prometheus user myself, but my understanding was that both sources of metrics could be collected as Prometheus metrics.  Is that accurate?

Reply:
Yes, that is accurate.

Reply:
I'll call them both out specifically